lib/src/platform/src/realtime/realtime.dart

```diff
@@ -54,58 +56,74 @@ class Realtime extends PlatformObject {
   /// Provides access to the underlying Connection object
   ///
   /// https://docs.ably.com/client-lib-development-guide/features/#RTC2
-  @override
   Connection get connection => _connection;
 
-  @override
-  Auth? auth;
+  /// a custom auth object to perform authentication related operations
+  /// based on the [options]
+  ///
+  /// https://docs.ably.com/client-lib-development-guide/features/#RSC5
+  // Auth? auth;
 
-  @override
-  ClientOptions options;
+  /// [ClientOptions] indicating authentication and other settings for this
+  /// instance to interact with ably service
+  late ClientOptions options;
 
-  @override
+  /// a push object interacting with Push API
+  /// viz., subscribing for push notifications, etc
```

Comment:
I think those comments belong to the class implementation itself and not needed here. (It's probably there already) Some for other dependencies in this class